.env

```
@@ -1,4 +1,5 @@
 NODE_ENV=development
 PORT=3000
 LOG_LEVEL=info
-COUCHDB_URL=http://localhost:5984
\ No newline at end of file
+COUCHDB_URL=http://localhost:5984
+BASE_URL=http://localhost
```

```suggestion
BASE_URL=http://localhost:3000
```

Addresses Mike's comment re. URL